Metadata/Driver/AnnotationDriver.php

```diff
@@ -175,7 +180,7 @@ public function loadMetadataForClass(\ReflectionClass $class)
                         $params[] = $this->convertReferenceValue($paramName, $annot->params[$paramName]);
                     }
 
-                    if (!$params) {
+                    if (!$params && !$annot instanceof Call) {
                         continue;
```

Comment:
``@AfterSetup`` doesn't work actually and just adds logic for something very similar to ``@InjectParams`` so I think that we should merge them to make it simpler.
This was the initial goal of ``@Call``.